opal/mca/event/libevent2022/libevent/poll.c

```diff
@@ -162,6 +162,9 @@ poll_dispatch(struct event_base *base, struct timeval *tv)
 
 	EVBASE_RELEASE_LOCK(base, th_base_lock);
 
+	if (msec > 0) {
+		msec = 0;
+	}
```

Comment:
As noted in the comments on this PR, this change to libevent is... tricky.  What does it mean for external libevents?

Reply:
reverted this